.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Bug report
+about: 'Help contribute with us by creating a bug report.'
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!-- This template should be used when creating issues, failing to do so might result in the report being closed. -->
```

Comment:
<!-- This template should be used when creating issues, failing to do so might result in your report being closed. -->